apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -82,20 +80,32 @@ export default function Rundown(props: RundownProps) {
   );
 
   // Handle keyboard shortcuts
-  const handleKeyPress = useCallback(
+  const handleKeyDown = useCallback(
     (event: KeyboardEvent) => {
       // handle held key
       if (event.repeat) return;
-      // Check if the alt key is pressed
-      if (event.altKey && (!event.ctrlKey || !event.shiftKey)) {
+
+      if (event.metaKey || event.ctrlKey) {
+        setEditMode(EditMode.CherryPick);
+
+        return;
```

Comment:
metaKey still uses the windows key, which is unfortunate
do you think it is sane to check for OS when handling metaKey? we have a small utility for it using UserAgent